<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-05-08T17:09:01Z" name="Default suite" finished-at="2021-05-08T17:09:49Z" duration-ms="47407">
    <groups>
    </groups>
    <test started-at="2021-05-08T17:09:01Z" name="Default test" finished-at="2021-05-08T17:09:49Z" duration-ms="47407">
      <class name="com.AutomationTask.demoProject1.TestCalss1">
        <test-method is-config="true" signature="launchDriver()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:01Z" name="launchDriver" finished-at="2021-05-08T17:09:12Z" duration-ms="10953" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="Test1()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:12Z" name="Test1" finished-at="2021-05-08T17:09:27Z" duration-ms="14960" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Welcome to New TESSAA] but found [Welcome to New TESSA]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Welcome to New TESSAA] but found [Welcome to New TESSA]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.AutomationTask.demoProject1.TestCalss1.Test1(TestCalss1.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:27Z" name="quit" finished-at="2021-05-08T17:09:29Z" duration-ms="1725" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method is-config="true" signature="launchDriver()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:29Z" name="launchDriver" finished-at="2021-05-08T17:09:44Z" duration-ms="14927" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="Test3()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:44Z" name="Test3" finished-at="2021-05-08T17:09:47Z" duration-ms="2984" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test3 -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.AutomationTask.demoProject1.TestCalss1@5677323c]" started-at="2021-05-08T17:09:47Z" name="quit" finished-at="2021-05-08T17:09:49Z" duration-ms="1523" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.AutomationTask.demoProject1.TestCalss1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
